app/src/src/components/common/FormatDate/MonthDayYear.jsx

```diff
@@ -0,0 +1,16 @@
+import YYYYMMDD from '../FormatDate/YYYYMMDD';
+// takes in Date and returns the date in Month Day Year format (ex: January 1 1960)
+const MonthDayYear = ({ date }) => {
+  let yyyymmdd = date;
+  // check if date is in Date format or is already in YYYY-MM-DD format (birthday saves in YYYY-MM-DD format)
+  const check = /^\d{4}-\d{2}-\d{2}$/;
+  if (!date.match(check)) yyyymmdd = YYYYMMDD({ selectedValue: date });
+  const months = 'January February March April May June July August September October November December'.split(
```

Comment:
Why use .split? Cant you just make the array yourself?

Reply:
sorry, I was being lazy. I'll fix it